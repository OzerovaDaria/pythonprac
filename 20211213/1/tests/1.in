[12555, -8898, 0, 0, -7, 676767, 1000000000, False, True, 0, 0, -1, -1, -2, 3, 4]
