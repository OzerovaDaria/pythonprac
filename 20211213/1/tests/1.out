[-8898, -7, -2, -1, -1, 0, 0, False, 0, 0, True, 3, 4, 12555, 676767, 1000000000]
